web/themes/custom/pineapple/sass/layouts/nodes/_article-teaser-page.scss

```diff
@@ -0,0 +1,51 @@
+.views-row {
+  margin-bottom: 16px;
+}
+
```

Comment:
This file should just be called article, no need to include display mode in the name

Reply:
done